shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/TargetReplication.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf;
+
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "target-replication")
```

Comment:
This matches the layout in the YAML right? Where it makes sense I think we should re-use the same names as we use in Circus Train, so I was thinking more like something like this:

```
    table-replications:
      - source-table: 
          database-name: foo
          table-name: bar
```
i.e. in the YAML file you can provide a list of tables to replicate. We could also allow regex to be used for both the db and table name and then compile this into a list of potential matches.